app/javascript/article-form/components/EditorBody.jsx

```diff
@@ -1,14 +1,61 @@
 import { h } from 'preact';
 import PropTypes from 'prop-types';
 import Textarea from 'preact-textarea-autosize';
+import { useEffect, useRef } from 'preact/hooks';
 import { Toolbar } from './Toolbar';
+import {
+  handleImageDrop,
+  handleImageFailure,
+  onDragOver,
+  onDragExit,
+} from './dragAndDropHelpers';
+import { useDragAndDrop } from '@utilities/dragAndDrop';
+
+function handleImageSuccess(textAreaRef) {
+  return function (response) {
+    // Function is within the component to be able to access
+    // textarea ref.
+    const editableBodyElement = textAreaRef.current.base;
+    const { links, image } = response;
+
+    const markdownImageLink = `![${image[0].name}](${links[0]})`;
```

Comment:
can we remove the extension from the `alt` text? So "screenshot" instead of "screenshot.png"

Reply:
Done.